GCS/probe0/probe0.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+import "github.com/google/cloudprober/probes/proto/config.proto";
+import "github.com/googleinterns/step224-2020/config/proto/probe.proto";
+option go_package = "github.com/googleinterns/step224-2020/GCS/probe0/proto";
+
+//The probe0 package is the package for the test probe - creates files in GCS with contents without any significance
+
+package probe0;
+
+message ProbeConf {
+  required string bucket = 1;
```

Comment:
Don't use `required` if you use proto2 syntax.
https://developers.google.com/protocol-buffers/docs/style#things_to_avoid

Reply:
Thank you changed to optional!